### DISCUSSION TITLE
Test Redis before before Enabling Drop-in

### DISCUSSION THREAD

**Discussion by jordantrizz** [rhubarbgroup/.github#5] [Ideas (ROC)]
There is no method to test to confirm if the Redis server is listening and is connectable before enabling the plugin's drop-in. It seems like a standard function that should be available; otherwise, one could down their site with a fatal error due to the drop-in not checking if Redis is enabled.

**Comment by tillkruss** [comment]
@jordantrizz: We're working on this right now. You can follow it here: https://github.com/rhubarbgroup/redis-cache/pull/464
